include/seqan3/io/alignment_file/format_sam_base.hpp

```diff
@@ -453,45 +453,43 @@ inline void format_sam_base::read_header(stream_view_type && stream_view,
         read_field(stream_view | views::take_until_or_throw(is_char<'\t'> || is_char<'\n'>), value);
     };
 
-    // @HQ line
```

Comment:
why removing this? Of course after fixing it to HD.

Reply:
1) Because I do not now if the header comes first 2) It is now documented in line 460